cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,15 @@
+const { generateUser } = require('../support/generate');
```

Comment:
please commit your support/generate.js file where we can see that you have implemented generateUser function